CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/AppShell.xaml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<Shell x:Class="CheckDrive.Mobile.AppShell"
-       xmlns="http://xamarin.com/schemas/2014/forms"
-       xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-       xmlns:views="clr-namespace:CheckDrive.Mobile.Views"
-       Title="CheckDrive.Mobile"
-       NavBarIsVisible="False">
+<Shell
+    x:Class="CheckDrive.Mobile.AppShell"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:views="clr-namespace:CheckDrive.Mobile.Views"
+    Title="CheckDrive.Mobile"
+    NavBarIsVisible="False">
```

Comment:
Update your formatter, we should all use the same formatting options, otherwise each change would be showing unnecessary changes in formatting which would be distracting the reviewer.

Reply:
but it works like this for me Format XAML, Source: [https://www.youtube.com/shorts/TCz2IDuNZE8](url)

Reply:
You should configure this tool in Visual Studio to start attributes on the same line as the element